star-wars-app/UI/View.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  View.swift
+//  star-wars-app
+//
+//  Created by Michał Nowak on 13/01/2020.
+//  Copyright © 2020 Michal Nowak. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class View {
+    // Base
+    static func notTranslatesAutoresizing<V: UIView>(_ view: V) -> Void {
```

Comment:
it doesn't need to be generic. Typesystem just needs to know, that it's UIView in order to be able to set this flag.

You can check out CommonStyles.swift file in ClassBuzz. I'm not saying it uses the best approach, but you're pretty incosistent with your styles. Some return styling functions and some are generic, but return nothing.